src/solvers/LevenbergMarquardt.jl

```diff
@@ -271,9 +281,9 @@ function step_solver!(
     get_residuals!(dmp, lms.candidate_residual_values, temp_x)
 
     ρk =
-        2 * (normFk2 - norm(lms.candidate_residual_values)^2) / (
+        (normFk2 - norm(lms.candidate_residual_values)^2) / (
```

Comment:
Is this this part?
![image](https://github.com/JuliaManifolds/Manopt.jl/assets/6612981/acb37632-d01d-4574-9e3c-bb81ea9a7fcd)
So should it include the `1/2`

Reply:
ρk is calculated as:
![image](https://github.com/JuliaManifolds/Manopt.jl/assets/6612981/37e76ec3-0a0c-47f1-8f5e-6c61147ade39)

Reply:
You're again correct, though this shouldn't impact convergence that much.

Reply:
Thanks, it was only the `^2` in ρk that seemed to make a difference in my example not converging.